### DISCUSSION TITLE
netem impairs interface, but no show output

### DISCUSSION THREAD

**Discussion by matthias2280** [srl-labs/containerlab#2152] [Q&A]
Ubuntu 24.04 host.  When I run:
sudo clab tools netem show -n NODE
I get output: 
Error: could not get all qdiscs: unmarshalNetem()       14
        [244 103 138 117 242 143 166 15]

This only happens after I set an impairment the first time.  On a node with no impairments set I get the expected table of interfaces and N/A for impairments.  As soon as I set an impairment, the table view breaks and I get the above output.  
The impairment works, i.e I can inject latency and packet loss, and I can remove and set impairments back to defaults, however the tabular view is broken until the lab is redeployed.

**Comment by hellt** [comment]
Hi
And what happens when you redeploy the lab? Do you see the set netem values? Or never?
